example/README.md

```diff
@@ -2,78 +2,50 @@ This is a new [**React Native**](https://reactnative.dev) project, bootstrapped
 
 # Getting Started
 
->**Note**: Make sure you have completed the [React Native - Environment Setup](https://reactnative.dev/docs/environment-setup) instructions till "Creating a new application" step, before proceeding.
+> **Note**: Make sure you have completed the [React Native - Environment Setup](https://reactnative.dev/docs/environment-setup) instructions till "Creating a new application" step, before proceeding.
 
-## Step 1: Start the Metro Server
+## NodeJS and Ruby
 
-First, you will need to start **Metro**, the JavaScript _bundler_ that ships _with_ React Native.
+To run the project you need to install the correct version of NodeJS and Ruby.
+We recommend the use of a virtual environment of your choice. For ease of use, this guide adopts [nodenv](https://github.com/nodenv/nodenv) for NodeJS, [rbenv](https://github.com/rbenv/rbenv) for Ruby.
 
-To start Metro, run the following command from the _root_ of your React Native project:
+The node version used in this project is stored in [.node-version](.node-version),
+while the version of Ruby is stored in [.ruby-version](.ruby-version).
+
+## Build the app
 
 ```bash
-# using npm
-npm start
+# Install NodeJS with nodenv, the returned version should match the one in the .node-version file
+$ nodenv install && nodenv version
 
-# OR using Yarn
-yarn start
-```
+# Install Ruby with rbenv, the returned version should match the one in the .ruby-version file
+$ rbenv install && rbenv version
 
-## Step 2: Start your Application
+# Install yarn and rehash to install shims
+$ npm install -g yarn && nodenv rehash
 
-Let Metro Bundler run in its _own_ terminal. Open a _new_ terminal from the _root_ of your React Native project. Run the following command to start your _Android_ or _iOS_ app:
+# Install bundle
+$ gem install bundle
 
-### For Android
+# Install the required Gems from the Gemfile
+# Run this only during the first setup and when Gems dependencies change
+$ bundle install
 
-```bash
-# using npm
-npm run android
+# Install dependencies
+# Run this only during the first setup and when JS dependencies change
+$ yarn install
 
-# OR using Yarn
-yarn android
+# Install podfiles when targeting iOS (ignore this step for Android)
+# Run this only during the first setup and when Pods dependencies change
+$ cd iOS && bundle exec pod install && cd ..
 ```
 
-### For iOS
+## Run the app
 
 ```bash
-# using npm
-npm run ios
+# Android
+$ yarn react-native run-android
 
-# OR using Yarn
-yarn ios
+# iOS
+$ yarn react-native run-ios
 ```
```

Comment:
What do you think to use 
`yarn ios` or `yarn android` from the example directory?

Reply:
Done, thank you.